x-pack/platform/plugins/private/upgrade_assistant/public/application/components/es_deprecations/deprecation_types/data_streams/flyout/steps/confirm/confirm_step.tsx

```diff
@@ -84,36 +85,75 @@ export const ConfirmReindexingFlyoutStep: React.FunctionComponent<
     }));
   };
 
+  const startActionButtonLabel =
+    resolutionType === 'reindex'
+      ? i18n.translate(
+          'xpack.upgradeAssistant.dataStream.migration.flyout.checklistStep.startActionButtonLabel',
+          {
+            defaultMessage: 'Start reindexing',
+          }
+        )
+      : i18n.translate(
+          'xpack.upgradeAssistant.dataStream.migration.flyout.checklistStep.startActionButtonLabel',
+          {
+            defaultMessage: 'Mark all read-only',
+          }
+        );
+
+  const actionClarification =
+    resolutionType === 'reindex' ? (
+      <>
+        <p>
+          <FormattedMessage
+            id="xpack.upgradeAssistant.dataStream.flyout.warningsStep.reindex.acceptChangesTitle"
+            defaultMessage="{count, plural, =1 {# backing index} other {# backing indices}}, including current write index, will be re-indexed. Current write index will be rolled over first."
+            values={{
+              count: meta.indicesRequiringUpgradeCount,
+            }}
+          />
+        </p>
+        <EuiSpacer size="s" />
+        <p>
+          <FormattedMessage
+            id="xpack.upgradeAssistant.dataStream.flyout.warningsStep.reindex.acceptChangesTitle"
+            defaultMessage="You can increase the speed of reindexing by changing throttling configuration on ES. Where changing throttling configuration allows you to utilize more resources to speed up the reindexing process. {learnMoreHtml}"
+            values={{
+              learnMoreHtml: (
+                <EuiLink
+                  href={`${links.elasticsearch.docsBase}data-stream-reindex-api.html#reindex-data-stream-api-settings`}
```

Comment:
>                  href={`${links.elasticsearch.docsBase}data-stream-reindex-api.html#reindex-data-stream-api-settings`}

Instead of putting the URL together like this, can you please create a new keyword in get_doc_links.ts with the full URL? This URL will break when we shift (imminently) to the new style of links for V9+

Reply:
This needs to be fixed in a separate PR since the fix needs to also be backported to `8.18` and `8.x`. I created an issue to tackle it next. I created the issue https://github.com/elastic/kibana/issues/215227 I requested another review for this PR bypassing this issue